opensrp-chw-core/src/test/java/org/smartregister/chw/core/utils/CoreConstantsTest.java

```diff
@@ -76,6 +76,223 @@ public void testGetFamilyRegister() {
         Assert.assertEquals(Utils.getLocalForm("family_register", locale, assetManager), CoreConstants.JSON_FORM.getFamilyRegister());
     }
 
+    @Test
+    public void testGetBirthCertification() {
```

Comment:
The test method should follow the format `test<Method Name><Condition><Expected>`. 

For this case for example, it could be something like `testGetBirthCertificationReturnsCorrectConstant`